pkg/controller/test-images-distributor/test_images_distributor.go

```diff
@@ -291,6 +323,20 @@ func (r *reconciler) reconcile(ctx context.Context, req reconcile.Request, log *
 	*log = *log.WithField("cluster", cluster).WithField("namespace", decoded.Namespace).WithField("name", decoded.Name)
 	log.Info("Starting reconciliation")
 
+	// One of the following is allowed:
+	// - multiarch namespaces to distribute on the proper non-amd64 clusters (ex.: ci-arm64 on arm01)
+	// or
+	// - non-multiarch namespaces to distribute on any amd64 clusters (ex.: ci on build03)
+	if isMultiarchNamespace(decoded.Namespace) {
```

Comment:
Why do we need to check both cases? If the namespace is multi-arch so be it, otherwise it is amd64 by default.

Reply:
Oh, that extra check will deny the distribution of the amd64 images to arm01... Nice.